Changes

```diff
@@ -1,5 +1,22 @@
 Release history of Zonemaster
 
+v2017.4.2 2018-02-23 Security release
+
```

Comment:
Shouldn't it be v2018.1?

As per https://github.com/dotse/zonemaster/blob/b461694b841121b2a0bf9a3b847fdb8d72f10857/docs/design/Versions%20and%20Releases.md
"Version Number Syntax for Zonemaster product"

Reply:
This is an update outside the plan. We have only patching of two components. We could make this v2018.1, but then we have to change the plan too.

Both solutions are OK for me.